src/content-script.ts

```diff
@@ -17,28 +17,23 @@ Object.defineProperty(HTMLScriptElement.prototype, 'src', {
     },
     set(this: HTMLScriptElement, val: string) {
         if (val.startsWith(public_path)) {
-            console.log('Hijacking', val)
+            console.debug('Hijacking', val)
+            const err = () => (this.onerror || (() => {})(), '')
             fetch(val)
-                .then(data => data.text())
-                .then(eval)
-            chunks.add(this)
+                .then(data => data.text(), err)
+                .then(eval, err)
```

Comment:
FYI, the current es/tslintrc hates `eval`. May be a good idea to add a comment to shut it up.